pkg/autoops/api/error.go

```diff
@@ -143,6 +143,10 @@ var (
 		codes.FailedPrecondition,
 		"autoops progressive rollout: can not use progressive rollout when feature is disabled",
 	)
+	statusProgressiveRolloutFeaturePrerequisiteExistss = gstatus.New(
+		codes.FailedPrecondition,
+		"autoops progressive rollout: can not use progressive rollout when feature has prerequisites",
```

Comment:
nit: `autoops: can not create a progressive rollout when the feature has prerequisites`